README.md

```diff
@@ -1,6 +1,34 @@
 # virtual_shake_robot_pybullet
 Virtual shake robot v2 based on PyBullet for earthquake studies
 
-Dependencies : 
+# Project  Overview:
+The Virtual Shake Robot (VSR) project is a sophisticated simulation tool designed to study the dynamics of Precariously Balanced Rocks (PBRs) during overturning and large-displacement processes. Utilizing the PyBullet physics engine, the VSR offers an innovative approach to understanding seismic hazards, rockfall prediction, and the fate of toppled PBRs with realistic material properties and terrains. This project is developed with the integration of advanced robotics and machine learning technologies, aiming to automate rock mapping and analysis effectively.
```

Comment:
The Virtual Shake Robot (VSR) V2.0 project is a simulation tool designed to study the dynamics of Precariously Balanced Rocks (PBRs) during overturning and large-displacement processes. Utilizing the PyBullet physics engine, the VSR offers an approach to study the effects of physics parameters on the dynamics of PBRs, which help us understand seismic hazards, rockfall prediction, and the fate of toppled PBRs with realistic material properties and terrains. This project is developed with the integration of robotics and machine learning technologies, aiming to automate rock mapping and analysis effectively.